tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -1013,7 +1155,12 @@ private void SetupSqliteConfigWithInvalidInstallCommand(string databaseName)
 
             // Use an in memory SQLite database
             // See https://www.sqlite.org/inmemorydb.html
+#if NETSTANDARD
+            var connectionString = "Data Source=:memory:";
```

Comment:
cool! Thanks for that!

Reply:
No prob :)

For Sqlite to fully work under net standard you need the package reference to the full Microsoft sqlite package and use some different syntax for the connection string.
Tripped me up quite a bit on one of my own projects just a week or two ago.